woocommerce/woocommerce-ios
# [Mobile Payments] Correct dark mode for connect image and fix section heading too

⚠️ This PR is against the feature branch implementing support for Card Present Payments, not against develop ⚠️

Fixes #3738 

Changes:
- Use PDF instead of SVG for great glory (and transparent background)
- Minimize unwanted section heading on connect screen
- All other background problems cited in #3738 were collaterally fixed by other changes

How to test:
- Enable Dark Mode
- Enter Settings
- Tap on Manage Card Reader
- Ensure all cells have not-quite-black background
- Ensure image of guy with reader doesn't have a white bounding box
- Ensure there is no black gap between the nav bar and the first cell
- Look at it in light mode too

Before:

<img src="https://user-images.githubusercontent.com/1595739/113620448-2c7d0a00-960f-11eb-90ef-6b7e0d34ece1.png" width=40%/>

After:

<img src="https://user-images.githubusercontent.com/1595739/113620464-3141be00-960f-11eb-9e5f-253b68fcd80b.png" width=40%/>

<img src="https://user-images.githubusercontent.com/1595739/113620462-30a92780-960f-11eb-8e25-d10cd57b41f2.png" width=40%/>

Update release notes:
